src/guides/anatomy-of-an-http-transaction.md

```diff
@@ -0,0 +1,429 @@
+  var method = request.method;
+  var url = request.url;
+  var body = '';
+  request.on('error', function(err) {
+    console.error(err);
+  }).on('data', function(chunk) {
+    body += chunk;
+  }).on('end', function() {
+    // At this point, we have the headers, method, url and body, and can now
+    // do whatever we need to in order to respond to this request.
+  });
+}).listen(8080); // Activates this server, listening on port 8080.
+```
+
+If we run this example, we'll be able to *receive* requests, but not *respond*
+to them. In fact, if you hit this example in a web browser, your request would
+time out, as nothing is being sent back to the client.
+
+So far we haven't touched on the `response` object at all, which is an instance
+of [`ServerResponse`][], which is a [`WritableStream`][]. It contains many
+useful methods for sending data back to the client. We'll cover that next.
+
+## HTTP Status Code
+
+If you don't bother setting it, the HTTP status code on a response will always
+be 200. Of course, not every HTTP response warrants this, and at some point
+you'll definitely want to send a different status code. To do that, you can set
+the `statusCode` property.
+
+```javascript
+response.statusCode = 404; // Tell the client that the resource wasn't found.
+```
+
+There are some other shortcuts to this, as we'll see soon.
+
+## Setting Response Headers
+
+Headers are set through a convenient method called [`setHeader`][].
+
+```javascript
+response.setHeader('Content-Type', 'application/json');
+response.setHeader('X-Powered-By', 'bacon');
+```
+
+When setting the headers on a response, the case is insensitive on their names.
+If you set a header repeatedly, the last value you set is the value that gets
+sent.
+
+## Explicitly Sending Header Data
+
+The methods of setting the headers and status code that we've already discussed
+assume that you're using "implicit headers". This means you're counting on node
+to send the headers for you at the correct time before you start sending body
+data.
+
+If you want, you can *explicitly* write the headers to the response stream.
+(Remember, the `response` object is a [`WritableStream`][].) To do this,
+there's a method called [`writeHead`][], which writes the status code and the
+headers to the stream.
+
+```javascript
+response.writeHead(200, {
+  'Content-Type': 'application/json',
+  'X-Powered-By': 'bacon'
+});
+```
+
+Once you've set the headers (either implicitly or explicitly), you're ready to
+start sending response data.
+
+## Sending Response Body
+
+Since the `response` object is a [`WritableStream`][], writing a response body
+out to the client is just a matter of using the usual stream methods.
+
+```javascript
+response.write('<html>');
+response.write('<body>');
+response.write('<h1>Hello, World!</h1>');
+response.write('</body>');
+response.write('</html>');
+response.end();
+```
+
+Remember, the `end` function on streams can also take in some optional data to
```

Comment:
Is `Remember, ` in the mode of `Keep in mind that`, or is it intended to cause the reader to recall previously stated information?